test/steps/test_prepare_config.py

```diff
@@ -1,5 +1,7 @@
 # coding=utf-8
 """Prepare the skos-history config file and folder structure feature tests."""
```

Comment:
not sure what happened in this file.

does it run?

Reply:
I restructured it to reflect the modifications I made in the `.feature` file a while ago.

Of course, it runs, I don't push code that fails the tests